esphome/components/display/display.cpp

```diff
@@ -511,6 +511,8 @@ void Display::do_update_() {
     this->page_->get_writer()(*this);
   } else if (this->writer_.has_value()) {
     (*this->writer_)(*this);
+  } else {
```

Comment:
When LVGL is in use, this would get triggered since there is no lambda.

```suggestion
  } else if (this->auto_clear_enabled_) {
```

Reply:
Or what Jesse said.

Reply:
Good point, i will follow Jesses suggestion. And make it a callable method.